.circleci/config.yml

```diff
@@ -107,3 +109,36 @@ workflows:
               ignore: /.*/
             tags:
               only: /^v.*/
+
+  "Orb Test and Publish":
+    jobs:
+      - orb-tools/lint:
+          name: "lint-orb-files"
+          lint-dir: orb
+          filters:
+            branches:
+              only: /.*/
+            tags:
+              only: /.*/
+      - orb-tools/pack:
+          name: "pack-orb"
+          source-dir: orb
+          requires:
+            - "lint-orb-files"
+          filters:
+            branches:
+              only: /.*/
+            tags:
+              only: /.*/
+      - orb-tools/publish:
+          name: "publish-dev-branch-orb"
+          orb-ref: reactiveops/rok8s-scripts@dev:${CIRCLE_BRANCH}
+          publish-token-variable: CIRCLECI_DEV_API_TOKEN
+          attach-workspace: true
```

Comment:
What does `attach-workspace` do? The docs aren't very detailed about this or I'm just looking in the wrong place.

Reply:
It allows us to pass the workspace directories from job to job without saving artifacts (at least I'm fairly sure that's what it does).  I pulled most of this workflow from the example orb-tools stuff.